Libraries/LibWeb/HTML/Parser/HTMLTokenizer.cpp

```diff
@@ -487,8 +487,8 @@ Optional<HTMLToken> HTMLTokenizer::next_token(StopAtInsertionPoint stop_at_inser
                 }
                 ON_EOF
                 {
-                    m_queued_tokens.enqueue(move(m_current_token));
-                    EMIT_EOF;
+                    m_current_token.set_comment(consume_current_builder());
+                    EMIT_CURRENT_TOKEN_FOLLOWED_BY_EOF;
```

Comment:
The spec text here says:
> Emit the comment. Emit an end-of-file token.

The spec text for the [comment start dash state](https://html.spec.whatwg.org/multipage/parsing.html#comment-start-dash-state), says:

> This is an [eof-in-comment](https://html.spec.whatwg.org/multipage/parsing.html#parse-error-eof-in-comment) [parse error](https://html.spec.whatwg.org/multipage/parsing.html#parse-errors). Emit the current comment token. Emit an end-of-file token.

This PR makes these 2 states behave the same way. Should there be a difference in behavior here? I'm not familiar enough with the spec language to know.

Reply:
I would take those to be equivalent.